hummingbot/templates/conf_pure_market_making_strategy_TEMPLATE.yml

```diff
@@ -63,5 +63,8 @@ jump_orders_enabled: null
 # If jump_orders_enabled is True, what is the depth in base currency to be used for finding top bid and ask
 jump_orders_depth: null
 
+# Parameter to enable/disable adding transaction costs to order prices
+add_transaction_costs: null
```

Comment:
When you modify the template file, please also increment the template number on top